lib/everypolitician/popolo/person.rb

```diff
@@ -5,7 +5,8 @@ class People < Collection; end
     class Person < Entity
       class Error < StandardError; end
 
-      attr_accessor :name, :email, :image, :gender, :birth_date, :death_date, :honorific_prefix, :honorific_suffix
+      attr_accessor :name, :email, :image, :gender, :birth_date, 
+                    :death_date, :honorific_prefix, :honorific_suffix
```

Comment:
If you're explicitly creating a method for this, you don't need to add it to here — this is just for methods we want to magically spring into existence without defining them